lib/globals.js

```diff
@@ -84,7 +84,12 @@ let globals = {
         }
         items = items.map(item => {
             if (typeof item === 'string') item = { value: item };
-            if (!item.text && !item.html && !item.label) {
+            if (item.label) {
```

Comment:
Should we have some tests for getItems() ?

Reply:
They're only being tested through the components at the moment, as that seemed to make more sense to check what the real-life output was. But the coverage is a little lacking.
the label html and text was theoretical, so i might remove this chunk anyway, which only leaves the placeholder code needing tests

Reply:
:+1: